### ISSUE TITLE
Add canonical_url to claim table

### ISSUE THREAD

**Issue by kauffj** [lbryio/chainquery#124]
The SDK returns this property on claim, can it be provided by chainquery as well?

**Comment by tiger5226** [comment]
Absolutely. I was going to add it a few months ago, but it was still being tweaked at the time. Do we have a specific description of it written down somewhere for me to reference? @tzarebczan maybe you have something on it.
